configs/example-config-extpipesv3.yml

```diff
@@ -5,7 +5,7 @@ extpipes:
     # description: string [ 1 .. 500 ] characters
 
     # extpipe-pattern only documentation atm
-    extpipe-pattern: '{source}:{short-name}:{rawtable-name}:{suffix}'
+    extpipe-pattern: '{source}:{short-name}:{table-name}:{suffix}'
```

Comment:
let's rename this from `extpipe-pattern`  to `naming-pattern` and think about how to use it to (optional) validate the configured names?